client/GameForm.js

```diff
@@ -9,12 +9,13 @@ import TextField from '@material-ui/core/TextField/TextField';
 import { useInput } from './hooks/InputHook';
 
 const GameForm = ({ history }) => {
-    const {value: listType, bind: bindListType} = useInput('collection');
-    const {value: listOption, bind: bindListOption} = useInput('');
-    const {value: numPlayers, bind: bindNumPlayers} = useInput('');
-    const {value: minDuration, bind: bindMinDuration} = useInput('');
-    const {value: maxDuration, bind: bindMaxDuration} = useInput('');
-   
+    const { value: listType, bind: bindListType } = useInput('collection');
+    const { value: listOption, bind: bindListOption } = useInput('');
+    const { value: numPlayers, bind: bindNumPlayers } = useInput('');
+    const { value: minDuration, bind: bindMinDuration } = useInput('');
+    const { value: maxDuration, bind: bindMaxDuration } = useInput('');
+    const durationInputProps = { min: '0', max: '1440', step: '5' }; 

```

Comment:
Just a thought, but it may be worth making the max for `minDuration` be the current value of `maxDuration` and the min for `maxDuration` be the current `minDuration` (in both cases, only if the other is present - ie: not `''`).

This would push the user away from a nonsensical edge-case.  I don't feel super strongly about it.